lib/tasks/servers.rake

```diff
@@ -10,16 +10,29 @@ task servers: :environment do
     puts("\tsecret: #{server.secret}")
     puts("\t#{server.enabled ? 'enabled' : 'disabled'}")
     puts("\tload: #{server.load.presence || 'unavailable'}")
+    puts("\tload multiplier: #{server.load_multiplier}")
```

Comment:
Can this be changed to `server.load_multiplier.nil? ? 1.0 : server.load_multiplier.to_d`

Reply:
Done on fd9dcf7